docs/auth/google/gcp-iap-auth.md

```diff
@@ -71,12 +72,10 @@ export default async function createPlugin(
             // Somehow compute the Backstage token claims. Just some dummy code
             // shown here, but you may want to query your LDAP server, or
             // GSuite or similar, based on the IAP token sub/email claims
-            const id = `user:default/${iapToken.email.split('@')[0]}`;
-            const fullEnt = ['group:default/team-name'];
-            const token = await ctx.tokenIssuer.issueToken({
-              claims: { sub: id, ent: fullEnt },
-            });
-            return { id, token };
+            const id = iapToken.email.split('@')[0];
+            const sub = stringifyEntityRef({ kind: 'User', name: id });
+            const ent = [sub, 'group:default/team-name'];
```

Comment:
Perhaps `stringifyEntityRef` this one too so we don't encourage this? Or we could drop it

Reply:
added